opencog/guile/SchemeSmobValue.cc

```diff
@@ -326,7 +335,7 @@ SCM SchemeSmob::ss_value_to_list (SCM svalue)
 	ValuePtr pa(verify_protom(svalue, "cog-value->list"));
 	Type t = pa->get_type();
 
-	if (nameserver().isA(t, FLOAT_VALUE))
+	if (nameserver().isA(t, FLOAT_VALUE) or nameserver().isA(t, OCTO_VALUE))
 	{
 		const std::vector<double>& v = FloatValueCast(pa)->value();
```

Comment:
If this works, then it only works by accident... because, currently, `OCTO_VALUE` does not actually inherit from `FLOAT_VALUE`.  See line 49 of `opencog/atoms/value/atom_types.script` It says:
```
OCTO_VALUE <- VALUE
```
You can change this if you want ... but the OctoValue C++ class should inherit the same way that the `atom_types.script` file says it does; otherwise, there's a problem.

Reply:
The C++ class inherits from FloatValue. I will modify the atom_types.script to reflect that.